feature_aggregation_scripts/ocean_subBasin_regions/ocean_subBasin_regions.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python
+
+import shapely
+import numpy
+import xarray
+import os
+import matplotlib.pyplot as plt
+import pyproj
+import gzip
+import shutil
+import tempfile
+
+from geometric_features import GeometricFeatures, FeatureCollection
+from geometric_features.feature_collection import _round_coords
+
+from geometric_features.download import download_files
+from geometric_features.utils import write_feature_names_and_tags
+
+
+def main():
+    author = 'Milena Veneziani'
+
+    # make a geometric features object that knows about the geometric data
+    # cache up a couple of directories
+    gf = GeometricFeatures(cacheLocation='../../geometric_data',
+                           remoteBranchOrTag='master')
```

Comment:
This should be an input argument because the calling code will usually already know what is needed.
```suggestion
```